File: Progamming 1/Lab9/Find_a_word_in_a_file_Java/FindAWord.java
import java.io.*;
import java.util.*;

public class FindAWord {

    public static boolean contains(String filename, String testWord) {
        //Make this method work!
        try {
            BufferedReader br = new BufferedReader(new FileReader(filename));

            while (br.ready()) {
                if(testWord.equals(br.readLine()) == true){
                    return true;
                }
            }

            br.close();
        }
        catch (FileNotFoundException e) {
            System.err.println("The file doesn't exist.");
        }
        catch (IOException e) {
            System.err.println("Something went wrong on the read.");
        }

        return false;
    }

    public static void main(String[] args) {
        
        if (contains(args[0], args[1])) {
            System.out.println("The file contains the word.");
        }
        else {
            System.out.println("The file does not contain the word.");
        }
    }
}

File: Progamming 1/Lab9/Find_a_word_in_a_file_Python/find_a_word.py
import sys


def contains(filename, test_word) -> bool:
    # complete this method!
    # test_word.trim
    # print(test_word)x
    with open(filename, 'r') as f: 
        for i in f: 
            if test_word in i:
                return True

    return False


if __name__ == "__main__":
    if contains(sys.argv[1], sys.argv[2]):
        print("The file contains the word.")
    else:
        print("The file does not contain the word.")
